test/compilable/jsonNoOutFile.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 set -euo pipefail
 dir=${RESULTS_DIR}/compilable
-output_file=${dir}/jsonNoOutFile.sh.out
-$DMD -c -od=${dir} -Xi=compilerInfo compilable/extra-files/emptymain.d > ${output_file}
-rm -f emptymain.json || true
+$DMD -c -od=${dir} -Xi=compilerInfo compilable/extra-files/emptymain.d
+rm -f emptymain.json
```

Comment:
Not sure why `|| true` was added here. `rm -f` always exits with `0` and also this tests doesn't check the generated output.

Reply:
Cause I didn't know `rm -f` always returned 0 :)

Reply:
I remember I was having troubles trying to check the generated output, I can't remember why though...